### PR TITLE
[FIX] Change some references to OpenERP - [IMP] Improve documentation to add an explanation of after migration

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
:+1: thanks!

File: openerp/openupgrade/doc/source/after_migration.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+After migration
+===============
+
+When migration through OpenUpgrade is completed, you may need to make another
+complete update (with --update=all parameter passed to the server) with last
+Odoo official version.
+
+OpenUpgrade keep old model tables and old columns (prefixed with
```
Comment (hbrunn) [inline openerp/openupgrade/doc/source/after_migration.rst]: [...] keepS [...]

File: openerp/openupgrade/doc/source/after_migration.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+`openupgrade_legacy_` word) in PostgreSQL as a conservative strategy because
+they can be used in a later migration script (or to fix an incorrect migration)
+or only for reference.
+
+After the upgraded database has been running stable for a while, you can
+consider remove this extra data for saving DB space.  There's a module called
```
Comment (hbrunn) [inline openerp/openupgrade/doc/source/after_migration.rst]: `consider to remove` or `consider removing`. And there's a double space

File: openerp/openupgrade/doc/source/intro.rst [hunk 3]
```diff
@@ -1,33 +1,31 @@
 Introduction
```
Comment (hbrunn) [inline openerp/openupgrade/doc/source/intro.rst:1]: is this a merge from upstream git-wise? if not, please do that

File: openerp/openupgrade/doc/source/after_migration.rst (hunk 2 above)
Comment (StefanRijnhart) [inline openerp/openupgrade/doc/source/after_migration.rst]: Oh, actually cleaning the database is also import to restore the consistency between the ORM and the database layout. The data is orphaned, which means there is no protection from the ORM when another module or a future version of Odoo reintroduces any table or column for a new purpose.

File: openerp/openupgrade/doc/source/intro.rst (hunk 3 above)
Comment (pedrobaeza) [inline openerp/openupgrade/doc/source/intro.rst:1]: I don't understand you. What do you mean?

**Comment by pedrobaeza** [comment]
I have fixed the comments except one from @hbrunn that I don't understand.

**Comment by legalsylvain** [comment]
:+1: Thanks.

File: openerp/openupgrade/doc/source/intro.rst (hunk 3 above)
Comment (hbrunn) [inline openerp/openupgrade/doc/source/intro.rst:1]: Ignore me, I though that was an update of odoo's doc, but it's OpenUpgrade

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**